docs/chianode.md

```diff
@@ -25,6 +25,19 @@ spec:
 
 If you would like to ensure your replicas get scheduled on different kubernetes nodes, view the [Pod Affinity documentation.](all.md#pod-affinity)
 
+## Full Node Peers
+
+You may optionally specify a list of full_nodes for peer(s) to sync your node from.
```

Comment:
```suggestion
You may optionally specify a list of full_nodes peer(s) that your node will always try to remain connected to
```

Technically the node will connect to other nodes as well, up to the configured peer limits, but for the full node, it will just ensure it is always connected to these nodes (as long as they are online)